drivers/lora/sx12xx_common.c

```diff
@@ -8,18 +8,23 @@
 #include <drivers/gpio.h>
 #include <drivers/lora.h>
 #include <logging/log.h>
+#include <sys/atomic_builtin.h>
 #include <zephyr.h>
 
 /* LoRaMac-node specific includes */
 #include <radio.h>
 
 #include "sx12xx_common.h"
 
+#define MODEM_CLAIM(data)       atomic_cas(&data.modem_usage, 0, 1)
```

Comment:
Should these be callsed MODEM_ACQUIRE/RELEASE or MODEM_LOCK/UNLOCK instead? I would normally assume that FOO_FREE means that the memory for FOO is deallocated.

Reply:
Yes